integrated_channels/sap_success_factors/models.py

```diff
@@ -120,10 +121,11 @@ class SAPSuccessFactorsEnterpriseCustomerConfiguration(EnterpriseCustomerPluginC
         blank=True,
         default='',
         verbose_name="Client Secret",
-        help_text=_("OAuth client secret.")
+        help_text=_("OAuth client secret."),
+        null=True
```

Comment:
Any reason to make it `null=True` now?

Reply:
@zamanafzal in the follow up PR we need to remove all references for these fields so while creating records of SAP config in the test cases if these are not set to be nullable removing references will cause errors.